README.md

```diff
@@ -5886,1100 +5947,1186 @@ A few macros are using recursion to work.
 
-##### void M\_MEMORY\_DEL (type *ptr)
+##### `void M_MEMORY_DEL (type *ptr)`
 
-Delete the cleared object pointed by the pointer 'ptr'
-that was previously allocated by the macro M\_MEMORY\_ALLOC.
-'ptr' can not be NULL.
+Delete the cleared object pointed by the pointer `ptr`
+that was previously allocated by the macro `M_MEMORY_ALLOC`.
+`ptr` can not be NULL.
 
-The default used function is the 'free' function of the LIBC.
+The default used function is the `free` function of the LIBC.
 
-##### type *M\_MEMORY\_REALLOC (type, ptr, number)
+##### `type *M_MEMORY_REALLOC (type, ptr, number)`
 
-Return a pointer to an array of 'number' objects of type 'type'
-'ptr' is either NULL (in which the array is allocated), 
-or a pointer returned from a previous call of M\_MEMORY\_REALLOC 
+Return a pointer to an array of `number` objects of type `type`
+`ptr` is either NULL (in which the array is allocated), 
+or a pointer returned from a previous call of `M_MEMORY_REALLOC` 
 (in which case the array is reallocated).
 The objects are not initialized, nor the state of previous objects changed
 (in case of reallocation).
-The address of the previous objects may have moved and the MOVE operator
+The address of the previous objects may have moved and the `MOVE` operator
 is not used in this case.
 In case of allocation error, it returns NULL.
 
-The default used function is the 'realloc' function of the LIBC.
+The default used function is the `realloc` function of the LIBC.
 
-##### void M\_MEMORY\_FREE (type *ptr)
+##### `void M_MEMORY_FREE (type *ptr)`
 
-Delete the cleared object pointed by the pointer 'ptr'.
-The pointer was previously allocated by the macro M\_MEMORY\_REALLOC.
-'ptr' can not be NULL.
+Delete the cleared object pointed by the pointer `ptr`.
+The pointer was previously allocated by the macro `M_MEMORY_REALLOC`.
+`ptr` can not be NULL.
 
-The default used function is the 'free' function of the LIBC.
+The default used function is the `free` function of the LIBC.
 
-##### void M\_MEMORY\_FULL (size\_t size)
+##### `void M_MEMORY_FULL (size_t size)`
 
 This macro is called by M\*LIB when a memory error has been detected.
-The parameter 'size' is what was tried to be allocated (as a hint).
-The macro can :
+The parameter `size` is what was tried to be allocated (as a hint).
+The macro can:
 
-* abort the execution,
-* throw an exception (In this case, the state of the object is unchanged),
-* set a global error variable and return.
+* Abort the execution
+* Throw an exception. In this case, the state of the object is unchanged
+* Set a global error variable and return
 
-NOTE: The last two cases are not properly fully supported yet.
-Throwing an exception is not fully supported yet 
-(Needs M\*LIB support to clear the skipped objects).
+> [!NOTE] 
+> The last two cases are not properly fully supported yet.
+> Throwing an exception is not fully supported yet
+> (Needs M\*LIB support to clear the skipped objects).
```

Comment:
The note is obsolete. Throwing an exception is now supported

The 3 last sentences should be : "The last case is not properly fully supported yet."